src/main/resources/templates/fragments/owners.html

```diff
@@ -122,9 +122,13 @@ <h2>Pets and Visits</h2>
           </tr>
           <tr>
             <td><a th:href="@{__${owner.id}__/pets/__${pet.id}__/edit}"
-                   th:attr="hx-get=@{__${owner.id}__/pets/__${pet.id}__/edit}" hx-target="#block-content">Edit Pet</a></td>
+                   hx:get="@{__${owner.id}__/pets/__${pet.id}__/edit}"
+                   hx:push-url="@{__${owner.id}__/pets/__${pet.id}__/edit}"
```

Comment:
You could replace this with `hx-push-url="true"` and it should work just the same.

Reply:
Done. I also replaced all the unnecessary `HttpServletRequest` and `HttpServletResponse` method parameters. I wonder if we even need the `HtmxResponse` (controller methods can return strings), but I didn't go that far.